tests/Group.test.ts

```diff
@@ -10,7 +10,8 @@ import { fromLine } from './TestHelpers';
 window.moment = moment;
 
 function checkGroupNamesOfTask(task: Task, property: GroupingProperty, expectedGroupNames: string[]) {
-    const group = Group.getGroupNamesForTask(property, task);
+    const grouping: Grouping = { property };
+    const group = Group.getGroupNamesForTask(grouping, task);
```

Comment:
Picking an arbitrary place to add a comment about a missing test...

CONTRIBUTING has a section on tests to add for new instructions: there is a place in Query.test.ts to add any new `sort` instructions, to ensure that they are correctly hooked up and parsed, please.